avm/res/network/virtual-network-gateway/main.json

```
@@ -5,8 +5,8 @@
   "metadata": {
     "_generator": {
       "name": "bicep",
-      "version": "0.23.1.45101",
-      "templateHash": "13396048573854621912"
+      "version": "0.24.24.22086",
+      "templateHash": "8829136319288976181"
```

```suggestion
      "version": "0.23.1.45101",
      "templateHash": "13396048573854621912"
```

Restoring as no new version needs to be published